demo/demo.js

```diff
@@ -58,27 +69,27 @@ const showTooltip = commit => {
                     </p>
                 </div>
             </div>
-        `)
-        .classed(direction, true)
-        .style({
-            left: `${left}px`,
-            top: (d3.event.pageY + 16) + 'px',
-        });
+        `);
+        tooltip.style('left', `${left}px`)
```

Comment:
You will need an extra package d3 multi . 
https://www.npmjs.com/package/d3-selection-multi .
for 2 lines of a code I didn't wanted to start up with that